tests/profile_speed.py

```diff
@@ -133,9 +140,11 @@ def convert_size(size_bytes):
         'system': uname.system,
         'machine': uname.machine,
         'processor': uname.processor,
+        'n_cpu': multiprocessing.cpu_count(),
         'memory': convert_size(psutil.virtual_memory().total),
         'n_soundscapes': n_soundscapes,        
-        'time_taken_for_script': time_taken,
+        'time_taken': np.round(time_taken, 4),
```

Comment:
why aren't we calling this runtime again?

Reply:
I thought this was clearer. Runtime could also be related to program execution (e.g. `RuntimeError`).

Reply:
how about `execution_time` then?